gcc/testsuite/gcc.target/riscv/cv-march-xcvalu-fail-compile-addn.c

```diff
@@ -0,0 +1,11 @@
+
+/* { dg-do compile } */
+/* { dg-require-effective-target cv_alu } */
+/* { dg-options "-march=rv32i_xcvalu1p0 -mabi=ilp32" } */
```

Comment:
Need to make sure we get rid of versioning? But that should come with the upcoming commits :)